server/src/config.rs

```diff
@@ -131,10 +130,18 @@ pub struct Config {
     pub grpc_port: u16,
     pub grpc_server_cq_count: usize,
     pub timeout: Option<ReadableDuration>,
-    /// Enable that use tenant as the schema if schema is not provided.
-    pub enable_tenant_as_schema: bool,
+    /// The batch size of the query response.
+    pub min_rows_per_query_batch: usize,
+    /// The threshold of the datums in the query response to trigger
+    /// compression.
+    pub query_response_size_compression_threshold: ReadableSize,
```

Comment:
Too long name, maybe just `compress_min_length`
- https://www.techrepublic.com/article/how-to-configure-gzip-compression-with-nginx/

Reply:
Ok. But I guess the name should indicate to be relative with query. How about `query_resp_compress_min_length`?